packages/frontend/src/components/MetricQueryData/UnderlyingDataResultsTable.tsx

```diff
@@ -76,6 +76,7 @@ const UnderlyingDataResultsTable: FC<{
                     show: true,
                 }}
                 cellContextMenu={CellContextMenu}
+                $padding={10}
```

Comment:
I'm little bit worried about changing `$padding` prop defaults because it's being used in different components not only on tables. 

let's ship this and if we observe something odd we can revisit

Reply:
Ok. I couldn't find it anywhere else so I thought it was ok. But Maybe I'm not looking in the right places :).